ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -183,107 +184,102 @@ def ndmg_dwi_worker(
     print("Rescaling b-vectors...")
     preproc.rescale_bvec(fbvec, bvec_scaled)
 
-    # Check orientation (dwi_prep)
-    dwi_prep, bvecs = gen_utils.reorient_dwi(dwi_prep, bvec_scaled, namer)
+    # Check orientation (eddy_corrected_data)
+    eddy_corrected_data, bvecs = gen_utils.reorient_dwi(
+        eddy_corrected_data, bvec_scaled, prep_dwi
+    )
 
     # Check dimensions
-    dwi_prep = gen_utils.match_target_vox_res(dwi_prep, vox_size, namer, sens="dwi")
+    eddy_corrected_data = gen_utils.match_target_vox_res(
+        eddy_corrected_data, vox_size, outdir, sens="dwi"
```

Comment:
`reorient_dwi` takes `prep_dwi` as a base directory (e.g., `sub-#/ses-#/dwi/preproc`), whereas
`match_target_vox_res` takes `outdir`. 

This is because `match_target_vox_res` is used multiple times, and sometimes on anatomical files -- but it's a little bit confusing I think.